docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -394,3 +394,113 @@ endpoint :code:`zenkotos3.com`:
     },
     (...)
 
+    Backblaze as a data backend
```

Comment:
Rendering, Minor: This should not be indented. If you take a look at the rendering, it's broken, and your whole doc bit is in the code block, along with the previous `restEndpoints` JSON example.

Reply:
Woups, you're right, this should look better when fix 👍